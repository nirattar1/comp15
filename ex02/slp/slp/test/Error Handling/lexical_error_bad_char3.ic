class Stam{
	// The question is

	int c@n;

	// we find them all???
}