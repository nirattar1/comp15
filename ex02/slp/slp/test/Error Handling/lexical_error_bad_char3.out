4: Lexical error: illegal character '@'